test/system/trader.js

```diff
@@ -131,11 +100,20 @@ class Trader {
       price,
     };
     return this.client.createOrder(order)
-      .then(() => {
-        console.log((side === 'B' ? 'Buy' : 'Sell'), 'Qauntity is', quantity, 'price is' , price, 'order placed', this.config.name);
-      }).catch(err => {
-        console.log(err);
-      });
+      .then(
+        () => {
+          if (options && options.expectFail) {
```

Comment:
unless we explicitly pass null in as the options argument, options will always be truthy, no need to check for it